### PR TITLE
Kata-CC: Upgrade to 0.6.3

### PR DESCRIPTION
**Pull request by dallasd1**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Kata-CC: Upgrade to 0.6.3

###### Change Log  

- Kata-CC: Upgrade to 0.6.3. A more detailed description can be found in the release notes

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Pipeline build id: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=480130&view=results

### CODE REVIEW

**Review by danmihai1 — APPROVED** [review]

File: SPECS/kata-containers-cc/kata-containers-cc.signatures.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
   "Signatures": {
-    "kata-containers-cc-0.6.2.tar.gz": "527a9e0064ba250e5355c03556ff60769a818aeec44481f77305d1fa115341b1",
-    "kata-containers-cc-0.6.2-cargo.tar.gz": "f43974d6f8305067d8dfd55edb3841fb8e7f1929405b9c4696b046b02c15bd76",
-    "mariner-coco-build-uvm.sh" :"2c1ef256c294c702ba2feab118644c81a2c6c85d0085fa8d205e3ce1a0b5c82d"
+    "mariner-coco-build-uvm.sh": "594c1e3764b65372a4fda2ab35fecaa2aa41937f37e0e3ccdf4778cc98f9b793",
+    "kata-containers-cc-0.6.3-cargo.tar.gz": "7ff6c5f7f7aa31a99ea5d837876291d886b16c32f21b6d65d044fd398abff1e6",
+    "kata-containers-cc-0.6.3.tar.gz": "1f366ce70bf83a239a7ec99334506adb28c3199157b4370840c3685378a34268"
   }
-}
+}
```
Comment (christopherco) [inline SPECS/kata-containers-cc/kata-containers-cc.signatures.json:7]: nit - keep newline at EOF
```suggestion
}

```

**Review by christopherco — APPROVED** [review]

File: SPECS/kata-containers-cc/kata-containers-cc.signatures.json (hunk 1 above)
Comment (dallasd1) [reply SPECS/kata-containers-cc/kata-containers-cc.signatures.json:7]: Will resolve this in the next patch to avoid re-running the pipelines

**Comment by PawelWMS** [comment]
Why are we pushing this change through the fast-track process instead of a regular release?

**Comment by dallasd1** [comment]
> Why are we pushing this change through the fast-track process instead of a regular release?

This is an MSRC issue and we're trying to get this in before its CVE is published

**Review by danmihai1 — APPROVED** [review]

**Review by christopherco — APPROVED** [review]

**Pull request merged**